main.go

```diff
@@ -2,16 +2,26 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"os"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
-	"os"
+)
+
+var (
+	VERSION   = "v0.11.0"
```

Comment:
1. I agree that it is easier to track than git commits, but not by much. I don't think it is so much easier that we need both.

2. Do we have a real world scenario where we actually build from non-version controlled source? And if that is an actual workflow, can we change that workflow to use version-controlled source instead?

I just don't like the idea of having to manually maintain version information inside the code. It is going to be a pain and lead to "Bumping version" commits or rebases or ammends.